wgpu-core/src/instance.rs

```diff
@@ -263,7 +263,7 @@ impl<B: GfxBackend> Adapter<B> {
             )))
             .optimal_tiling;
 
-        let mut allowed_usages = wgt::TextureUsage::empty();
+        let mut allowed_usages = wgt::TextureUsage::COPY_SRC | wgt::TextureUsage::COPY_DST;
```

Comment:
Can we initialize with the base texture usage that is guaranteed (i.e. `guaranteed_format_features.allowed_usages`), instead of this hard-coded value?